src/scripts.js

```diff
@@ -1,9 +1,38 @@
 // import './css/base.scss';
 import './css/styles.scss';
 
-import recipeData from './data/recipes';
-import ingredientsData from './data/ingredients';
-import users from './data/users';
+// import recipeData from './data/recipes';
+// import ingredientsData from './data/ingredients';
+// import users from './data/users';
+
+let users = [{}];
+let serverRecipeData = [{}];
+let ingredientsData = [{}];
+
+fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData')
```

Comment:
To answers your question, I might look into if "nested" fetch requests exist, but currently this is looking pretty good to me.

Reply:
I figure nesting them would dictate the order in which they are fired.